### PR TITLE
Search configMap with active profile name.

### PR DESCRIPTION
**Pull request by shahbour**
Based on #153 this pull request will search and fetch configMap names with active profiles.

fixes gh-153

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -111,6 +111,9 @@ spring:
 In the example above, it `spring.cloud.kubernetes.config.namespace` had not been set,
 then the ConfigMap named `c1` would be looked up in the namespace that the application runs
 
+If a profile is active then it will also be looked at as `name-profile`. For example if `production` profile
+is active in the above example a `configMap` with name `c1-production` will fetched.
```
Comment (pukkaone) [inline README.adoc]: This [README.adoc file was generated](https://github.com/spring-cloud/spring-cloud-kubernetes/blob/master/README.adoc#documentation). Your change will be lost when the file is generated again. You should change the [documentation source files](https://github.com/spring-cloud/spring-cloud-kubernetes/tree/master/docs/src/main/asciidoc) instead.

**Comment by geoand** [comment]
This looks good to me, thanks @shahbour! 

@salaboy @spencergibb @ryanjbaxter WDYT?

**Comment by spencergibb** [comment]
Can you rebase on the 1.0.x branch so we can get this in Greenwich.

**Comment by shahbour** [comment]
> Can you rebase on the 1.0.x branch so we can get this in Greenwich.

You are asking me to do that?

**Comment by ryanjbaxter** [comment]
@shahbour yes if you submit against the 1.0.x branch we can then merge it forward into master

**Comment by shahbour** [comment]
I did change the request to be into spring-cloud:1.0.x

**Pull request merged**